azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -193,18 +196,22 @@ public AzkabanWebServer() throws Exception {
    * Constructor
    */
   public AzkabanWebServer(Server server, Props props) throws Exception {
-    this.props = props;
+    this.props = requireNonNull(props);
```

Comment:
I think that's a good idea. btw can you point me to the annotation that you are referring to? I'm unable to find the annotation in our current classpath.